File: src/ui/src/api/models/vehicle-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { Location } from './location';
import { VehicleType } from './vehicle-type';
export interface VehicleViewModel {
  id?: number;
  lastKnownLocation?: Location;
  name?: null | string;
  type?: VehicleType;
}

File: src/vehicles/Fleet.Vehicles/ViewModels/VehicleViewModel.cs
using Fleet.Common;
using Fleet.Vehicles.Models;

namespace Fleet.Vehicles.ViewModels
{
    public class VehicleViewModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public VehicleType Type { get; set; }
        public Location? LastKnownLocation { get; set; }
    }
}
